src/components/Header/Header.js

```diff
@@ -27,38 +35,81 @@ function Header() {
   const greeting = "Hi There!";
 
   // all sidebar links lead to path: /
-  const SidebarData = [
-    {
-      title: "Take the Challenge",
-      path: "/",
-      icon: <GiIcons.GiJeweledChalice />,
-    },
-    {
-      title: "Take Action",
-      path: "/",
-      icon: <GiIcons.GiJeweledChalice />,
-    },
-    {
-      title: "Notifications",
-      path: "/",
-      icon: <GiIcons.GiJeweledChalice />,
-    },
-    {
-      title: "Share",
-      path: "/",
-      icon: <GiIcons.GiJeweledChalice />,
-    },
-    {
-      title: "Why 8by8?",
-      path: "/",
-      icon: <GiIcons.GiJeweledChalice />,
-    },
-    {
-      title: "FAQS",
-      path: "/",
-      icon: <GiIcons.GiJeweledChalice />,
-    },
-  ];
+  var SidebarData;
+  var flagForTestNonChallenger=0;// make it non 0 if want to test the case for User Type is not challenger
+  if(getUserType()=="challenger" & flagForTestNonChallenger==0){
+
+    SidebarData = [
+      {
+        title: "My Challenge",
+        path: "/",
```

Comment:
All of these lead to the homepage "/" (placeholders from before)
I think we have most of the pages built out now and can start filling it out, ex. "My Challenge" is /progress